automatic/sqlite/update.ps1

```diff
@@ -5,50 +5,50 @@ $releases = 'https://sqlite.org/download.html'
 function global:au_SearchReplace {
    @{
         ".\legal\VERIFICATION.txt" = @{
-          "(?i)(\s+x32:).*"            = "`${1} $($Latest.URL32)"
-          "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
-          "(?i)(\s+Toolsx32:).*"       = "`${1} $($Latest.URLTools32)"
-          "(?i)(checksum32:).*"        = "`${1} $($Latest.Checksum32)"
-          "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
-          "(?i)(checksumTools32:).*"   = "`${1} $($Latest.ChecksumTools32)"
+          "(?<=x32:\s).*"             = "$($Latest.URL32)"
+          "(?<=\sx64:\s).*"           = "$($Latest.URL64)"
+          "(?<=\sToolsx64:\s).*"      = "$($Latest.URLTools64)"
+          "(?<=checksum32:).*"        = "$($Latest.Checksum32)"
+          "(?<=checksum64:).*"        = "$($Latest.Checksum64)"
+          "(?<=checksumTools64:\s).*" = "$($Latest.ChecksumTools64)"
```

Comment:
I don't see any reason to change the regular expressions here at all, except for 32 to 64.  To maintain consistency, and simplicity, can you please keep them in line with what is used elsewhere?

Reply:
Reverted to use standard regex approach.